vm/src/stdlib/json.rs

```diff
@@ -17,8 +19,23 @@ pub fn json_dump(obj: PyObjectRef, fs: PyObjectRef, vm: &VirtualMachine) -> PyRe
 }
 
 /// Implement json.loads
-pub fn json_loads(string: PyStringRef, vm: &VirtualMachine) -> PyResult {
+pub fn json_loads(obj: PyObjectRef, vm: &VirtualMachine) -> PyResult {
     // TODO: Implement non-trivial deserialization case
+    let string = if objtype::isinstance(&obj, &vm.ctx.str_type()) {
+        objstr::get_value(&obj)
+    } else if objtype::isinstance(&obj, &vm.ctx.bytearray_type())
+        || objtype::isinstance(&obj, &vm.ctx.bytes_type())
+    {
+        let bytes = objbyteinner::try_as_byte(&obj).unwrap();
```

Comment:
You can use serde_json::Deserializer::from_slice instead of converting to a string in this case.